scripts/generate-cli-completions.py

```
@@ -0,0 +1,419 @@
+    out = [
+        c
+        for c in registry
+        if not c.startswith("__") and c not in INTERNAL_COMMANDS
+    ]
+    return sorted(set(out))
+
+
+def help_commands_region(help_text: str) -> list[str]:
+    lines = help_text.splitlines()
+    try:
+        start = next(i for i, l in enumerate(lines) if l.strip() == "Commands:")
```

Ruff (0.15.18)

[error] Ambiguous variable name: `l`

(E741)